jdaviz/configs/imviz/plugins/parsers.py

```diff
@@ -216,7 +216,7 @@ def _parse_image(app, file_obj, data_label, ext=None, parent=None):
             # NOTE: if extending this beyond GWCS, the mouseover logic
             # for outside_*_bounding_box should also be updated.
             data.coords._orig_bounding_box = data.coords.bounding_box
```

Comment:
@bmorris3  - is this still needed/used anywhere?  Can we remove the entire if-statement?